nucypher/characters/lawful.py

```diff
@@ -702,7 +703,7 @@ def __init__(self,
                  checksum_address: ChecksumAddress = None,
                  operator_address: ChecksumAddress = None,  # TODO: deprecate, and rename to "checksum_address"
                  client_password: str = None,
-                 decentralized_identity_evidence=NOT_SIGNED,
+                 decentralized_identity_evidence_from_metadata=NOT_SIGNED,
```

Comment:
Why so wordy?

Reply:
Couldn't think of anything better, to be honest, and decided that explicit is at least clear. Suggestions?

Reply:
Kind of a tough one.  How about simply  `identity_evidence` or `wallet_signature`?

Reply:
Ah, I thought you're referring to the `_from_metadata` prefix. If you want to rename the whole thing, it'll have to be renamed in `nucypher-core` as well. `identity_evidence` is fine, or perhaps `operator_signature`?

Reply:
Either of those seem fine to me 👍🏻

Reply:
I did not want to make too many changes in this PR, only focusing on bringing it up to speed with `nucypher-core`, but I guess I can rename that everywhere. The `nucypher-core` renaming will have to wait the next release though.

Reply:
Renamed to `operator_signature`, filed https://github.com/nucypher/nucypher-core/issues/6